lang/c++/CMakeLists.txt

```diff
@@ -40,17 +40,21 @@ set (AVRO_VERSION_MINOR "0")
 project (Avro-cpp)
 set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${CMAKE_SOURCE_DIR})
 
-add_definitions(-std=c++11 -fPIC)
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
This change isn't working in our Docker images (and thus Travis)

Reply:
An older version of CMake perhaps? I think those appeared just after 3.0.

I'll take them out and make an update.

Reply:
Yep.  Our docker images use cmake 3.0.2, these were added in 3.1.